db/migrate/20170704165711_add_user_bio_and_token.rb

```diff
@@ -0,0 +1,26 @@
+class AddUserBioAndToken < ActiveRecord::Migration
+  def up
+    add_column :rusers, :bio, :text, limit: 2147483647, default: ''
```

Comment:
it's odd; the error I'm seeing is:

```
Error: test_confirm_user_reset_key_not_visible_on_profile_to_non-admins(UsersControllerTest): ActionView::Template::Error: wrong argument type nil (expected String)
app/views/users/profile.html.erb:149:in `to_html'
```

that's on this line:

```ruby
<p><small><%= raw auto_link(RDiscount.new(@profile_user.bio).to_html, :sanitize => false) %></small></p>
```

And `@profile_user` is a `User` as of this line: https://github.com/jywarren/plots2/blob/user-bio-and-token/app/controllers/users_controller.rb#L107

So, given this default, it should have a value of '', not `nil`. What's the deal?